File: ClusterMasterService/src/main/java/ase/cm/EventProcessing/Models/IoTResponse.java
package ase.cm.EventProcessing.Models;

/**
 * Created by Tommi on 14.06.2017.
 */
public class IoTResponse {
    public IoTResponseStatus getStatus() {
        return status;
    }

    public void setStatus(IoTResponseStatus status) {
        this.status = status;
    }

    public long getIdentifier() {
        return identifier;
    }

    public void setIdentifier(long identifier) {
        this.identifier = identifier;
    }

    public enum IoTResponseStatus{
        OK,
        FAIL
    };
    private IoTResponseStatus status;

    private long identifier;



}

File: Simulator/src/IoTResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ASE_2017_Simulator
{
    class IoTResponse
    {
        public enum IoTResponseStatus
        {
            OK,FAIL
        }

        public IoTResponseStatus status { get; set; }
        public long identifier { get; set; }
    }
}
